docs/ref/contrib/postgres/aggregates.txt

```diff
@@ -208,6 +208,31 @@ General-purpose aggregation functions
 
         Examples are the same as for :attr:`ArrayAgg.ordering`.
 
+    Usage example::
+
+        class Publication(models.Model):
+            title = models.CharField(max_length=30)
+
+        class Article(model.Model):
```

Comment:
```suggestion
        class Article(models.Model):
```

Reply:
Thanks! That's what I get from copy pasting from the JSONBAgg function, I have corrected that typo as well.